Index: src/main/java/hcmuaf/edu/vn/BikeEcommerce/Test.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package hcmuaf.edu.vn.BikeEcommerce;\r\n\r\nimport hcmuaf.edu.vn.BikeEcommerce.model.Category;\r\nimport hcmuaf.edu.vn.BikeEcommerce.model.Product;\r\nimport hcmuaf.edu.vn.BikeEcommerce.service.BrandService;\r\nimport hcmuaf.edu.vn.BikeEcommerce.service.CategoryService;\r\nimport hcmuaf.edu.vn.BikeEcommerce.service.ProductService;\r\n\r\npublic class Test {\r\n    public static void main(String[] args) {\r\n\r\n        Product product = new Product(\"1\", \"Pega\", 4500000, \"Pega\",\"24 inch\", \"Thép\", \"1 năm\", 1, 1, null, null, null, null, 1 );\r\n        Category category = new Category(\"1\", \"Xe đạp thể thao\", \"chay em\", null, null, 1,1);\r\n        ProductService.getInstance().insert(product);\r\n        CategoryService.getInstance().insert(category);\r\n        ProductService.getInstance().update(product);\r\n        System.out.println(product);\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/hcmuaf/edu/vn/BikeEcommerce/Test.java b/src/main/java/hcmuaf/edu/vn/BikeEcommerce/Test.java
--- a/src/main/java/hcmuaf/edu/vn/BikeEcommerce/Test.java	(revision aad72cf53a146747469e156dc6e8451d4bd21594)
+++ b/src/main/java/hcmuaf/edu/vn/BikeEcommerce/Test.java	(date 1692242114681)
@@ -1,19 +1,10 @@
 package hcmuaf.edu.vn.BikeEcommerce;
 
-import hcmuaf.edu.vn.BikeEcommerce.model.Category;
-import hcmuaf.edu.vn.BikeEcommerce.model.Product;
-import hcmuaf.edu.vn.BikeEcommerce.service.BrandService;
-import hcmuaf.edu.vn.BikeEcommerce.service.CategoryService;
-import hcmuaf.edu.vn.BikeEcommerce.service.ProductService;
+import hcmuaf.edu.vn.BikeEcommerce.service.UserService;
 
-public class Test {
+public class test {
     public static void main(String[] args) {
-
-        Product product = new Product("1", "Pega", 4500000, "Pega","24 inch", "Thép", "1 năm", 1, 1, null, null, null, null, 1 );
-        Category category = new Category("1", "Xe đạp thể thao", "chay em", null, null, 1,1);
-        ProductService.getInstance().insert(product);
-        CategoryService.getInstance().insert(category);
-        ProductService.getInstance().update(product);
-        System.out.println(product);
+        UserService userService = new UserService();
+        System.out.println(userService.getUserByKey("h@gmail.com"));
     }
 }
